doc/wiki/How-to-Contribute.md

```diff
@@ -1,3 +1,3 @@
 TBF.
 
-- [[ Update Wiki | Contribute:-Update-Wiki ]]
+- [[ Update Wiki | Contribute:‐Update‐Wiki ]]
```

Comment:
You are using a [hyphen](https://en.wikipedia.org/wiki/Hyphen#%22Unicode_hyphen%22) (`\u2010`) here, also in the file name `Contribute:‐Update‐Wiki.md`
The original one and all the other places use the [hyphen-minus](https://en.wikipedia.org/wiki/Hyphen-minus) (`\u002D`).

You can check them in the command line:
`grep -r "$(printf '\u2010')" How-to-Contribute.md` and `grep -r "$(printf '\u002D')" How-to-Contribute.md`

That's probably why it looks like this in your wiki:
![Screenshot from 2024-06-21 11-36-17](https://github.com/gamer-project/gamer/assets/24509818/f03260fd-cca6-4411-9dfa-731d96842b78)

Reply:
Thanks for pointing out. I have fixed it.